packages/contracts/test/verifier.test.ts

```diff
@@ -56,10 +61,16 @@ describe('Epoch key lite proof verifier', function () {
     this.timeout(300000)
 
     let unirepContract
+    let epochKeyLiteProofVerifier
+    let epochKeyProofVerifier
 
     before(async () => {
         const accounts = await ethers.getSigners()
         unirepContract = await deployUnirep(accounts[0])
+        epochKeyLiteProofVerifier = await deployEpochKeyLiteProofVerifier(
```

Comment:
The verifier contracts are decoupled from the unirep contract in this PR. There's no need to deploy `unirepContract` in these tests. You can make proofs with arbitrary inputs to test the verifiers.

This means you won't use the `UserState` object, you'll directly use `defaultProver` with whatever inputs you want.

Reply:
In this case, how would you prove that an attester state tree root exists without having access to the `unirepContract` AttesterData

Reply:
We're decoupling the verifier and unirep contract. In the epoch key contract for example we only need to verify that it correctly decodes the control output and returns the variables from the public signals. We don't actually care if the state tree root is valid, just that it's correctly parsed from the proof. State root validity is checked in the other tests.

The verifier isn't responsible for checking the state root, just for returning it.